aggregate.go

```diff
@@ -105,9 +135,21 @@ func (a *Aggregator) bundlesConflictWithDependencies(b1, b2 Bundle) bool {
 	return false
 }
 
-func (a *Aggregator) containsAggregatedBundle(list [][]Bundle, item []Bundle) bool {
+func (a *Aggregator) containsAggregatedBundle(list [][]Bundle, newGroup []Bundle) bool {
+	if len(newGroup) == 0 {
+		a.logger.Log("level", "warning", "message", "release aggregation observed empty list of version bundles")
+		return false
+	}
```

Comment:
This should actually never happen and there was no verification so far. Now being a bit more defensive and informative if shit hits the fan.